statchatui/src/Home/Home.jsx

```diff
@@ -11,8 +11,12 @@ const Home = ({ onStatsChange, stats }) => {
     platform: "",
     gameName: "",
   });
+
+  const [isLoading, setIsLoading] = useState(false);
+
   const handleSubmit = async (event) => {
     event.preventDefault();
+    setIsLoading(true); //sets the state to true before the api call is displaying the stats
```

Comment:
You don't really need a comment here since it's pretty self-explanatory! Same with the other comments below

Reply:
ok i will remove the comments